src/index.ts

```diff
@@ -1,4 +1,7 @@
 import VernaForm from './VernaForm';
-import { VernaContextProvider, useVerna } from './VernaContextProvider';
+import { VernaContextProvider, useVerna } from './context/VernaContextProvider';
+import WidgetWrapper from './WidgetWrapper';
+import type ShowCaseWidgetProps from './types/Widgets';
```

Comment:
Hmm... Not sure you should export types from here. I don't have the solution right now but this is something we have to address.